### DISCUSSION TITLE
Air Gap Install Fails

### DISCUSSION THREAD

**Discussion by baylorex97** [Security-Onion-Solutions/securityonion#5551] [Unsupported Versions] [answered]
Trying to install 2.7.3 using the iso downloaded from the SO site.  The server is a physical box with no internet access.  The install fails at 26%.  Seems like it is trying to reach external url's, which would fail.  Attach is the setup log

Thanks,
Chris

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/7182078/sosetup.log)

**Comment by dougburks** [comment, marked as answer]
Looking at your sosetup.log, this looks strange:
```
rsync: change_dir "/home/root/SecurityOnion/pillar" failed: No such file or directory (2)
```

Normally, you install the ISO, reboot, and login with a non-root user which then automatically starts Setup using sudo. Did that not happen here?

**Reply by baylorex97** [reply]
Thank you very much.  Looks like the ISO failed to install but continued with the SO install.  I started a complete reinstall.  It took a couple times, but the ISO finally completed.  The SO install ran fine.  Thanks again.
